### PR TITLE
add: star command in shortcuts

### PR DESCRIPTION
**Pull request by yihong0618**
@jdneo @zry656565 
Support star/unstar in codelen shortcuts/
image

### CODE REVIEW

**Comment by yihong0618** [comment]
Support #598 #549

**Comment by jdneo** [comment]
Thanks @yihong0618

I would suggest to use `$(star-empty)` and `$(star-full)`. You can find the style here: https://microsoft.github.io/vscode-codicons/dist/codicon.html

**Comment by yihong0618** [comment]
@jdneo 
Seems a bit low then the text title(Submit, Test)?
And is it necessary to change heart icon to star icon too?
image

**Comment by jdneo** [comment]
Ok, given the current rendering result, seems using text is better. But I think there is no need to put it as one of the default value

**Comment by yihong0618** [comment]
> Ok, given the current rendering result, seems using text is better. But I think there is no need to put it as one of the default value

Maybe in setting that user can config it open it or not?

**Comment by jdneo** [comment]
Do you mean using a setting to toggle between text and icon?

**Comment by yihong0618** [comment]
> Do you mean using a setting to toggle between text and icon?

Forget it, I think for now, keep the text maybe a better way.

**Comment by jdneo** [comment]
I think so too.

File: package.json [hunk 1]
```diff
@@ -621,7 +621,8 @@
                         "type": "array",
                         "default": [
                             "submit",
-                            "test"
+                            "test",
+                            "star"
```
Comment (jdneo) [inline package.json:625]: No need to add this shortcut by default. But please add it in the items, and also documents in readme

**Comment by yihong0618** [comment]
@jdneo  Fixed.

File: README.md [hunk 2]
```diff
@@ -90,9 +90,10 @@ Thanks for [@yihong0618](https://github.com/yihong0618) provided a workaround wh
   <img src="https://raw.githubusercontent.com/LeetCode-OpenSource/vscode-leetcode/master/docs/imgs/shortcuts.png" alt="Editor Shortcuts" />
 </p>
 
-- The extension supports 4 editor shortcuts (aka Code Lens):
+- The extension supports 5 editor shortcuts (aka Code Lens):
   - `Submit`: Submit your answer to LeetCode.
   - `Test`: Test your answer with customized test cases.
+  - `Star/Unstar`: Star or Unstar the current problem.
```
Comment (jdneo) [inline README.md:96]: nit: unstar

File: package.json [hunk 3]
```diff
@@ -629,6 +629,7 @@
                             "enum": [
```
Comment (jdneo) [inline package.json:629]: Could you by the way add `"enumDescriptions"` for this setting here? 😄

Comment (yihong0618) [reply package.json:629]: Fixed, and optimize star logic.

**Pull request merged**